keyboards/adpenrose/shisaku/info.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please redo the layout without rotation.
![image](https://user-images.githubusercontent.com/4781841/178653053-44648148-03ec-4d41-8eb4-e1de18202652.png)